pages/guide.py

```diff
@@ -206,7 +209,12 @@ class FeatureEditHandler(basehandlers.FlaskHandler):
       ('finch_url', 'link'),
       ('experiment_goals', 'str'),
       ('experiment_risks', 'str'),
-      ('experiment_extension_reason', 'str')]
+      ('experiment_extension_reason', 'str'),
+      ('rollout_milestone', 'int'),
+      ('rollout_platforms', 'split_str'),
+      ('rollout_details', 'str'),
+      ('enterprise_policies', 'split_str'),
+      ]
 
   CHECKBOX_FIELDS: frozenset[str] = frozenset([
```

Comment:
Isn't breaking_change also needed here?

Reply:
To be more clear, I think it is needed here.  If found that your code worked without including breaking_change in CHECKBOX_FIELDS, that is because of a different defect that was fixed a few days ago.